src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+function findSelectedTab(tabs, selectedTabId) {
+  return tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+}
```

Comment:
Move this function to a separate file. As I wrote above👆, you can reuse this function in another file